chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1943,7 +1942,7 @@ async def get_coin_records(coin_ids: Collection[bytes32]) -> List[CoinRecord]:
         assert e.code == expected
 
 
-TEST_FILL_RATE_ITEM_COST = 144_785_330
+TEST_FILL_RATE_ITEM_COST = 144_744_040
```

Comment:
the fact that this decreased also suggests that you had some "compression effects", doesn't it? Why else did it drop?
If you spend two coins that have the same parent coin ID, that 32 byte field will be compressed, even if the puzzle reveals are different

Reply:
It's documented that we use a different (parent) coin and a different puzzle, but we cannot completely eliminate the effect of compression, so we still have a small remaining effect.